### ISSUE TITLE
The RepositoryHandler.jcenter() method has been deprecated.

### ISSUE THREAD

**Issue by xlcrr** [react-native-async-storage/async-storage#610]
```
./gradlew --warning-mode all 

> Configure project :react-native-async-storage_async-storage
The RepositoryHandler.jcenter() method has been deprecated. This is scheduled to be removed in Gradle 8.0. JFrog announced JCenter's shutdown in February 2021. Use mavenCentral() instead.
```

react 16.13.1
react-native 0.63.4
 "@react-native-async-storage/async-storage": "^1.15.4"

gradle v7.0.2
buildToolsVersion = "30.0.2"
compileSdkVersion = 29

dependencies {
        classpath('com.android.tools.build:gradle:4.2.1')
}

**Comment by Krizzu** [comment]
Thanks, I'll keep this issue open as a reminder

**Comment by tomasswood** [comment]
Any update on this getting released @Krizzu ?

**Comment by tido64** [comment]
According to https://developer.android.com/studio/build/jcenter-migration, JCenter will allow downloads of existing artifacts indefinitely. This means that it's no longer urgent to move off JCenter. We will migrate whenever `react-native` does.

**Comment by donatoaguirre24** [comment]
@tido64 React native already did it 🙂

**Comment by wontone18** [comment]
Pls update your build.gradlew with mavenCentral()


build.gradlew

[repositories {
	mavenLocal()
    mavenCentral()
    maven {
        // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
        url "${resolveModulePath("react-native")}/android"
    }
    google()
    //jcenter()
}



repositories {
        google()
		mavenCentral()
       // jcenter()
    }](url)

**Comment by tido64** [comment]
Unfortunately, we cannot remove JCenter without breaking people on older versions of React Native:
```
Could not determine the dependencies of task ':app:lintVitalRelease'.
> Could not resolve all artifacts for configuration ':app:debugCompileClasspath'.
   > Could not find com.facebook.flipper:flipper:0.54.0.
     Required by:
         project :app
   > Could not find com.facebook.flipper:flipper-fresco-plugin:0.54.0.
     Required by:
         project :app
   > Could not find com.facebook.flipper:flipper-network-plugin:0.54.0.
     Required by:
         project :app
   > Could not find com.facebook.yoga:proguard-annotations:1.14.1.
     Required by:
         project :app > com.facebook.react:react-native:0.63.4
   > Could not find com.facebook.fbjni:fbjni-java-only:0.0.3.
     Required by:
         project :app > com.facebook.react:react-native:0.63.4
```
Like I mentioned earlier, there is no urgency here since JCenter will be up in read-only mode indefinitely.

**Comment by wontone18** [comment]
react native already updated even several other popular plugins they already updated their package. i can list out here

1. react native bootsplash also updated
2. react navigation also update.

except the above package i can send you a screen shot of their updated package. they are not using jcentre

**Comment by tido64** [comment]
It doesn't matter if they didn't also upload older versions so that react-native 0.63 and older can use Maven. If you look at the output I posted earlier, `com.facebook.fbjni:fbjni-java-only` 0.0.3 [doesn't exist on Maven](https://mvnrepository.com/artifact/com.facebook.fbjni/fbjni-java-only). It only has 0.0.4 and up. If we remove JCenter, people on react-native 0.63 and older cannot use AsyncStorage. Gradle 8 is pretty far out, so there is no immediate need to remove JCenter now. This is just a warning. Your app will still build.

**Comment by wontone18** [comment]
**then for a newer project asyncstorage we cant use it. current react native version is   "react-native": "0.65.1",**

**Comment by tido64** [comment]
I think there's been a misunderstanding here. If you're saying that we need to _add_ `mavenCentral()`, then I agree that we should do that. We don't have to remove `jcenter()` to do that.

**Comment by wontone18** [comment]
> I think there's been a misunderstanding here. If you're saying that we need to _add_ `mavenCentral()`, then I agree that we should do that. We don't have to remove `jcenter()` to do that.

:) i can understand what you are saying in the above. But with  any new project of react native 0.65 we cant use this package. for a moments.

**Comment by wibimaster** [comment]
@wontone18 
You can create a PR with the jcenter() removal, and we can install this project with `[org]/[repo]#[pull-id]/head`
This doesn't touch the main stream, and for people who want to get rid of the warning, it should be OK :)

**Comment by tido64** [comment]
Resolved by #697
